src/components/DetailsAlerts/DetailsAlerts.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
```

Comment:
Add copyrights text above, same as in all .js files

Reply:
Done